### ISSUE TITLE
tinytex fails to install lm-math when needed

### ISSUE THREAD

**Issue by bwiernik** [yihui/tinytex#90]
Using tinytex in RStudio 1.2.1194 with R 3.5.2 on Windows 10. I try to run this code with a fresh install of tinytex:
```
install.packages("psychmeta")
library(psychmeta)
ma_r_obj <- ma_r(ma_method = "ic", rxyi = rxyi, n = n, rxx = rxxi, ryy = ryyi,
                 construct_x = x_name, construct_y = y_name,
                 moderators = moderator, data = data_r_meas_multi)
metabulate(ma_obj = ma_r_obj, file = "meta tables correlations",
           output_format = "pdf", output_dir = tempdir())
```

It fails because it can't find the font "latinmodern-math". If I manually run `tlmgr_install(pkgs = "lm-math")`, the code above runs correctly.

**Comment by yihui** [comment]
This has already been fixed in the dev version two months ago via 660b5f06c939119f80314d93cade6dd97db0e55c. Please try `remotes::install_github('yihui/tinytex')`.

**Comment by bwiernik** [comment]
Yes, it works! Thanks, and sorry for the unnecessary report!
